modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -4,15 +4,9 @@
 
 import * as runtime from '../runtime';
 {{#imports.0}}
-import {
-    {{#imports}}
-    {{className}},
-    {{^withoutRuntimeChecks}}
-    {{className}}FromJSON,
-    {{className}}ToJSON,
-    {{/withoutRuntimeChecks}}
-    {{/imports}}
-} from '../models';
+// Some imports not used depending on template conditions
+// @ts-ignore
```

Comment:
**suggestion (non-blocking):** Looks like [TS docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#suppress-errors-in-ts-files-using--ts-ignore-comments) recommend adding the explanatory description in the same line, right after the `ts-ignore` term.
Probably not a big deal - but I imagine this could make those comments less intrusive

Reply:
Ok. I just copied it from typescript-axios. But I'll do it like the docs recommend.